.github/workflows/python.yml

```
@@ -0,0 +1,62 @@
+name: Python CI Workflow
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  build:
+    name: '[Python ${{ matrix.python-version }}]'
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version:
+          - 3.8
+
+    steps:
+
+      - uses: actions/checkout@master
+      - run: git fetch --prune --unshallow
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install Python tools
+        run: pip install setuptools_scm pytest
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+              git build-essential cmake cmake-extras libace-dev coinor-libipopt-dev libeigen3-dev swig \
+              qtbase5-dev qtdeclarative5-dev qtmultimedia5-dev libqt5charts5-dev \
+              libxml2-dev liboctave-dev python-dev python3-numpy valgrind libassimp-dev
```

Given that this should just build the Python bindings, I would limit the dependencies to the one strictly necessary.
```suggestion
          sudo apt-get install -y \
              git build-essential cmake cmake-extras libace-dev coinor-libipopt-dev libeigen3-dev swig \
              libxml2-dev python3-numpy libassimp-dev
```

---

Sure, I'll try to reduced them to the minimal ones.

---

I think b3e4667 addresses the requirement on minimal dependencies. Unfortunately `setuptools_scm` is not compatible with PEP51{7,8), and it must be installed in the system otherwise the proper version is not computed from the git tags resulting in `0.0.0` which, as a consequence, creates a faulty sdist package.